contrib/download_prerequisites

```diff
@@ -221,9 +225,6 @@ case $chksum_extension in
   ;;
 esac
 
-[ -e ./gcc/BASE-VER ]                                                         \
-    || die "You must run this script in the top-level GCC source directory"
-
```

Comment:
Could you please specify in the commit message why this was removed?

Reply:
Sorry, this is an unintended change :( Reverted,t hanks!